content/en/blog/2022/announcing-opentelemetry-demo-release.md

```diff
@@ -0,0 +1,67 @@
+
+One of our primary goals of this project has been to create a robust sample
+application for developers to use in learning OpenTelemetry, and we’re proud to
+say that we’ve done just that.
+Every OpenTelemetry language SDK is
+[represented](https://github.com/open-telemetry/opentelemetry-demo/blob/main/docs/service_table.md)
+in this release, with the exception of Swift -- yes, even PHP. 
+We’ve built complete [tracing flows](https://github.com/open-telemetry/opentelemetry-demo/blob/main/docs/trace_service_features.md) that demonstrate a breadth of common instrumentation tasks such as:
+
+- Enriching spans from automatic instrumentation.
+- Creating custom spans for richer, more useful traces.
+- Propagating trace context automatically and manually.
+- Handling observability baggage.
+- Creating attributes, events, and other telemetry metadata.
+
+We’ve also integrated OpenTelemetry Metrics across [several services](https://github.com/open-telemetry/opentelemetry-demo/blob/main/docs/metric_service_features.md) to capture runtime and business metric use cases.
+
+Now, it’d be enough to just provide a great demonstration of OpenTelemetry, but
+one thing we wanted to focus on for our 1.0 release was showing not just the
+‘how’, but the ‘why’, of OpenTelemetry.
+To that end, we’ve built a framework for implementing [failure
+scenarios](https://github.com/open-telemetry/opentelemetry-demo/blob/main/docs/README.md#scenarios)
+gated by feature flags.
+In addition, we include pre-configured dashboards and walk-thrus on how to read and interpret the telemetry data each service emits to discover the underlying cause of performance regressions in the application.
+
+Another goal of this demo is to streamline the ability of vendors and commercial
+implementers of OpenTelemetry to have a standardized target for building demos
+around.
+We’ve already seen quite a bit of adoption, with five companies including
+Datadog, Dynatrace, Honeycomb, Lightstep, and New Relic integrating the
+community demo application into their product demos (you can find a list
+[here](https://github.com/open-telemetry/opentelemetry-demo#demos-featuring-online-boutique)).
+We hope to encourage further contributions and collaboration along these lines.
+
+However, just because we reached 1.0, that doesn’t mean we’re stopping -- this
+demo is a living artifact, one that we intend to continue to improve. In the
+coming months we plan to continue to iterate and improve coverage of metrics and
+logs as more SDKs reach maturity.
+
+We also hope to add new instrumentation scenarios and patterns by extending the functionality of the application -- queues and async processing of requests, a hosted version in order to explore the demo with zero setup, adding in support for Swift, and more.
+
+We’d love for you to take the demo for a spin and let us know what you think!
+Check out the
+[docs](https://github.com/open-telemetry/opentelemetry-demo/tree/main/docs), or
```

Comment:
any plans to have the docs at https://opentelemetry.io/community/demo/?

Reply:
Eventually!